fastlane/Fastfile-deploy

```diff
@@ -0,0 +1,145 @@
+podspecFile = "MenigaSDK.podspec"
+
+platform :ios do
+
+  desc %(Deploy MenigaSDK to cocoapods trunk
+  Steps:
+   * Create compiled framework
+   * Bump version in podspec
+   * Commit framework and podspec changes
+   * Create version tag
+   * Push commit and tag to remote
+   * Push podspec to Trunk specs
+  Options:  
+  - version - new version number to release
+  )
+  lane :deploy do |options|
+    if !options[:version]
+      raise "No version specified!".red
+    end
+    version = options[:version]
+
+    build_framework
+    update_podspec_version(
+      version: version
+    )
+    create_deploy_commit(
+      version: version
+    )
+    push_to_git_remote
+    push_podspec
+  end
+
+  desc %(Deploy MenigaSDK to cocoapods trunk, 
+  lane developed to run on release branch from CI server
+  Steps:
+   * Create compiled framework
+   * Bump podspec version
+   * Commit framework and podspec changes as MenigaBuildsson
+   * Create version tag
+   * Push commit and tag to remote
+   * Push podspec to Trunk specs
+   * Merge branch into master
+  Options:  
+  - gh_user - Username fotr github account 
+  - gh_user_email - Email that be use to create commit 
+  - gh_token - Access token for gh_user_name github account 
+  )
+  lane :ci_deploy do |options|
+    branch_name = git_branch
+    if !branch_name.match("release/*")
+      raise "ci_deploy should be run only from release branch!".red
+    end
+
+    if !options[:gh_user]
+      raise "No username specified!".red
```

Comment:
`gh_user not specified`? I wonder if the message `No username specified` is readable enough.